Tasks/SonarQubePostTest/PRCA/PostComments-Module.psm1

```diff
@@ -83,6 +146,18 @@ function GetArtifactUri
     return $artifactUri
 }
 
+function ValidateComments
```

Comment:
The input comments are of type PSObject, which is more or less like Object, plus a few properties - so it's worth checking those properties exist.